extension/src/plots/model/index.ts

```diff
@@ -498,26 +457,36 @@ export class PlotsModel extends ModelWithPersistence {
     return this.commitRevisions[label] || label
   }
 
-  private getPlots(
-    checkpointPlots: CheckpointPlot[],
-    selectedExperiments: string[]
-  ) {
-    return reorderObjectList<CheckpointPlotData>(
-      this.metricOrder,
-      checkpointPlots.map(plot => {
-        const { id, values } = plot
-        return {
-          id,
-          title: truncateVerticalTitle(
-            id,
-            this.getNbItemsPerRow(Section.CHECKPOINT_PLOTS)
-          ) as string,
-          values: values.filter(value =>
-            selectedExperiments.includes(value.group)
-          )
-        }
-      }),
-      'id'
+  private getCustomPlotData(
+    plots: CustomPlot[],
+    colors: ColorScale | undefined
+  ): CustomPlotData[] {
+    if (!colors) {
+      return plots
+        .filter(plot => !isCheckpointPlot(plot))
+        .map(
```

Comment:
This function can definitely be simplified and only use one `map` instead of two.